### PR TITLE
doc: using pyvista.DynamicScraper()

### LINKED ISSUE #162: Dynamic examples
Generating dynamic examples in the gallery.

### PR DESCRIPTION
**Pull request by clatapie**
Closes #162.

### CODE REVIEW

**Comment by clatapie** [comment]
@ansys-reviewer-bot review

File: README.rst [hunk 1]
```diff
@@ -51,12 +51,12 @@ Install the latest package for use with this command:
...
 
 Alternatively, install the latest
-`PyAnsys Math GitHub <https://github.com/ansys/pyansys-math.git>`_ package
+`PyAnsys Math GitHub <https://github.com/ansys/pyansys-math>`_ package
 with this command:
 
 .. code::
 
-   pip install git+https://github.com/ansys/pyansys-math.git
+   pip install git+https://github.com/ansys/pyansys-math
```
Comment (clatapie) [inline README.rst:59]:
```suggestion
   pip install git+https://github.com/ansys/pyansys-math.git
```

**Comment by clatapie** [comment]
@ansys-reviewer-bot stop review

**Comment by clatapie** [comment]
The DynamicScraper renders correctly:
image

To have access to the rendering, the use of the `python -m http.server` command is necessary as explained in this [comment](https://github.com/ansys-internal/pyacp/pull/247#issue-1832853471).

File: doc/source/conf.py [hunk 2]
```diff
@@ -14,16 +15,14 @@
...
-pyvista.OFF_SCREEN = True
-
 # must be less than or equal to the XVFB window size
 try:
-    pyvista.global_theme.window_size = np.array([1024, 768])
+    pyvista.global_theme.window_size = np.array([1024, 768]) * 2
 except AttributeError:
     # for compatibility with pyvista < 0.40
-    pyvista.rcParams["window_size"] = np.array([1024, 768])
+    pyvista.rcParams["window_size"] = np.array([1024, 768]) * 2
```
Comment (germa89) [inline doc/source/conf.py:23]: why the number 2??

File: pyproject.toml [hunk 3]
```diff
@@ -53,16 +54,17 @@ doc = [
     "jupyter_sphinx==0.4.0",
     "jupyterlab==4.0.5",
     "numpydoc==1.5.0",
-    "pandas==2.1.0",
```
Comment (germa89) [inline pyproject.toml:56]: You dont depend on pandas anymore??  But yes on scipy? just double checking.

**Review by germa89 — APPROVED** [review]
It looks good to me!

File: pyproject.toml (hunk 3 above)
Comment (clatapie) [reply pyproject.toml:56]: `scipy` is used in the examples.
`pandas` in not used in `ansys-math-core`.

File: doc/source/conf.py (hunk 2 above)
Comment (clatapie) [reply doc/source/conf.py:23]: No particular reason. I will remove it.

**Pull request merged**